### ISSUE TITLE
No accessible names - "Congratulations to capital..." + "Read more of the news" links

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#77]
This element has no HTML markup that would provide an accessible name for the element. A screen reader would either read nothing for this element or may make a guess.
